package.json

```diff
@@ -57,7 +61,11 @@
     "virtualenv": "pip install virtualenv && virtualenv env && source env/bin/activate || . env/bin/activate && npm run pip",
     "pip": "pip install -r config/requirements.txt",
     "config": "cp config/secrets.py.example config/secrets.py",
-    "project-update": "pip install --upgrade pip && npm update"
+    "project-update": "pip install --upgrade pip && npm update",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "{webcompat/static/js/lib, tests, grunt-tasks}/**/*.js": "lint"
```

Comment:
The task is not fired up as the path seems not fitting, which is weird.
Let's make things easier and just switch from `"{webcompat/static/js/lib, tests, grunt-tasks}/**/*.js": "lint"` to `"*.js": "lint"`

So we are always on the safe side?